contract/Lykke.Service.BlockchainWallets.Contract/Models/BlockchainWalletsErrorResponse.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using Lykke.Common.Api.Contract.Responses;
+
+namespace Lykke.Service.BlockchainWallets.Contract.Models
+{
+    public class BlockchainWalletsErrorResponse : ErrorResponse
+    {
+        public ErrorCodeType CodeType { get; set; }
```

Comment:
```ErrorCodeType``` sounds like "type of the code of the error", while it's just a code of the error. So ```ErrorCode``` or ```ErrorType``` will be more suitable name. The same for the ```CodeType```, ```Code``` will be enough, or to be consistent with ```ErrorResponse.ErrorMessage``` property - ```ErrorCode```